f5_openstack_agent/lbaasv2/drivers/bigip/resource_manager.py

```diff
@@ -1042,7 +1042,16 @@ def delete(self, resource, service, **kwargs):
                 name=urllib.quote(payload['name']),
```

Comment:
If the member does not exist, will 404 is raised when loading?

Reply:
looks like  including load of pool & member into the try block is fine.